[2025-01-05T05:53:29.793+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-05T05:53:29.809+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available manual__2025-01-05T05:53:24.262265+00:00 [queued]>
[2025-01-05T05:53:29.817+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available manual__2025-01-05T05:53:24.262265+00:00 [queued]>
[2025-01-05T05:53:29.818+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-05T05:53:29.831+0000] {taskinstance.py:2889} INFO - Executing <Task(HttpSensor): is_api_available> on 2025-01-05 05:53:24.262265+00:00
[2025-01-05T05:53:29.840+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=469) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-05T05:53:29.841+0000] {standard_task_runner.py:72} INFO - Started process 471 to run task
[2025-01-05T05:53:29.842+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2025-01-05T05:53:24.262265+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpfkrx2eoc']
[2025-01-05T05:53:29.843+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask is_api_available
[2025-01-05T05:53:29.887+0000] {task_command.py:467} INFO - Running <TaskInstance: user_processing.is_api_available manual__2025-01-05T05:53:24.262265+00:00 [running]> on host a67ecbd59b00
[2025-01-05T05:53:29.973+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='is_api_available' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T05:53:24.262265+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T05:53:24.262265+00:00'
[2025-01-05T05:53:29.975+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-05T05:53:29.976+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-05T05:53:29.976+0000] {logging_mixin.py:190} INFO - Current task name:is_api_available state:running start_date:2025-01-05 05:53:29.810485+00:00
[2025-01-05T05:53:29.977+0000] {logging_mixin.py:190} INFO - Dag name:user_processing and current dag run status:running
[2025-01-05T05:53:29.978+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-05T05:53:29.978+0000] {baseoperator.py:416} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-01-05T05:53:29.979+0000] {http.py:137} INFO - Poking: api/
[2025-01-05T05:53:29.990+0000] {base.py:84} INFO - Retrieving connection 'user_api'
[2025-01-05T05:53:33.135+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-01-05T05:53:33.154+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-05T05:53:33.158+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, run_id=manual__2025-01-05T05:53:24.262265+00:00, execution_date=20250105T055324, start_date=20250105T055329, end_date=20250105T055333
[2025-01-05T05:53:33.176+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-05T05:53:33.177+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-05T05:53:33.179+0000] {logging_mixin.py:190} INFO - Dag name:user_processing queued_at:2025-01-05 05:53:24.277972+00:00
[2025-01-05T05:53:33.180+0000] {logging_mixin.py:190} INFO - Task hostname:a67ecbd59b00 operator:HttpSensor
[2025-01-05T05:53:33.194+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-05T05:53:33.243+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-05T05:53:33.247+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
